test/ably_event_listener_test.dart

```diff
@@ -25,15 +25,15 @@ class MockEmitter {
   int streamsCount;
   Map<int, int> indexes = {};
   List<int> injectables;
-  List<Stream<int>> streams;
+  late List<Stream<int>> streams;
```

Comment:
This doesn't need to be late, since we can just use `List<Stream<int>> streams = [];`

Reply:
It can also be marked final.

ff2a1ce4cc50d6cb75664e8256a083a28ae3445c